src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.cs

```diff
@@ -54,6 +54,10 @@ public static IEnumerable<object[]> Create_DifferentTfms()
             yield return new object[] { "build_property.TargetFramework = net5.0-ios14.0", true };
```

Comment:
Looking at the failures you'll either need to parse this in tests and generate identifier and version based on that or add them everywhere since we're likely skipping most of the warnings now because they're missing in the tests